### DISCUSSION TITLE
2021-05-19 Weekly Meeting 4 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#34] [Weekly Meetings]
Weekly Meeting 4 is happening on Wednesday 5/19/2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- Recap of the discussion around community and incentives for participating in OMI
- Discuss how to work XRSI's Diversity and Inclusion goals into our Code of Conduct / OKRs
- Reviewing proposals
  - [OMI Identity](https://github.com/omigroup/OMI/issues/36)
  - make sure to comment below with links to your proposals so we can get time scheduled to talk about them

**Comment by mrmetaverse** [comment]
Original submission from Kavya: https://github.com/omigroup/OMI/discussions/10#discussioncomment-750286 

Moving to set up a sync next Friday as a team to openly discuss XRSI!
